### PR TITLE
Add seq

### PR DESCRIPTION
**Pull request by SrilekhaKarnala**
Created a new Java file named `Sequence.java` and Implemented basic game functionality.

### CODE REVIEW

File: Seq.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+/*
+Alice and Bob are playing a game on a sequence a1,a2,...,an of length n . They move in turns and Alice moves first.  In the turn of each player, he or she should select an integer and remove it from the sequence. The game ends when there is no integer left in the sequence.  Alice wins if the sum of her selected integers is even; otherwise, Bob wins.  Your task is to determine who will win the game, if both players play optimally.
+*/
+import java.util.*;
+public class Seq{
+    public static void main(String args[])
+    {
+        Scanner sc=new Scanner(System.in);
+        int n=sc.nextInt();
+        int[] a=new int[n];
```
Comment (Jinankhan) [inline Seq.java]: nit: give better name to the array

File: Seq.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    
+/*
+Alice and Bob are playing a game on a sequence a1,a2,...,an of length n . They move in turns and Alice moves first.  In the turn of each player, he or she should select an integer and remove it from the sequence. The game ends when there is no integer left in the sequence.  Alice wins if the sum of her selected integers is even; otherwise, Bob wins.  Your task is to determine who will win the game, if both players play optimally.
+*/
+import java.util.*;
+public class Seq{
+    public static void main(String args[])
+    {
+        Scanner sc=new Scanner(System.in);
+        int n=sc.nextInt();
```
Comment (Jinankhan) [inline Seq.java]: make sure the variable names are descriptive

Comment (Jinankhan) [inline Seq.java]: edge case : what if the use enters a character which is not a valid integer?

File: Seq.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        int[] a=new int[n];
+        int codd=0;
+        for(int i=0;i<n;i++)
+        {
+            a[i]=sc.nextInt();
+            if(a[i]%2!=0)
+            codd++;
+        }
+        if(codd==0 || codd%2!=0)
```
Comment (Jinankhan) [inline Seq.java]: add this logic  in a ternary condition

File: Seq.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+Alice and Bob are playing a game on a sequence a1,a2,...,an of length n . They move in turns and Alice moves first.  In the turn of each player, he or she should select an integer and remove it from the sequence. The game ends when there is no integer left in the sequence.  Alice wins if the sum of her selected integers is even; otherwise, Bob wins.  Your task is to determine who will win the game, if both players play optimally.
+*/
+import java.util.*;
+public class Seq{
+    public static void main(String args[])
+    {
+        Scanner sc=new Scanner(System.in);
+        int n=sc.nextInt();
+        int[] a=new int[n];
+        int codd=0;
```
Comment (Jinankhan) [inline Seq.java]: descriptive variable names!

File: Seq.java [hunk 5]
```diff
@@ -0,0 +1,39 @@
+
+    
+/*
+Alice and Bob are playing a game on a sequence a1,a2,...,an of length n . They move in turns and Alice moves first.  In the turn of each player, he or she should select an integer and remove it from the sequence. The game ends when there is no integer left in the sequence.  Alice wins if the sum of her selected integers is even; otherwise, Bob wins.  Your task is to determine who will win the game, if both players play optimally.
+*/
+import java.util.*;
+public class Seq{
```
Comment (Jinankhan) [inline Seq.java]: Class name should be descriptive too

**Review by Jinankhan — CHANGES_REQUESTED** [review]

**Review by Jinankhan — APPROVED** [review]

**Comment by SrilekhaKarnala** [comment]
Merge Branch1

**Pull request merged**